paper/paper.md

```diff
@@ -94,46 +94,35 @@ bibliography: paper.bib
 
 # Summary
 
-With the emergence of the Brain Imaging Data Structure
-(``BIDS``; [@Gorgolewski2016]) in 2016, the neuroscientific community received
-a standard to organize and share data. Originally developed for Magnetic Resonance Imaging (MRI)
-data formats, BIDS now also supports magnetoencephalography (MEG) [@Niso2018],
-electroencephalography (EEG) [@Pernet2019], and intracranial EEG (iEEG) [@Holdgraf2019].
+With the development of the Brain Imaging Data Structure
+(``BIDS``; [@Gorgolewski2016]), the neuroscientific community received
+a standard to organize and share data. BIDS standardizes storage of complex experimental
+data by prescribing a set of file naming conventions and templates to store the
+associated metadata in the form of Javascript Object Notation (JSON) files. It thus
+facilitates data sharing, eases metadata querying and enables automatic data
+analysis pipelines.
 
-BIDS standardizes how complex experimental data should be structured and which
-metadata should be present besides the unprocessed data. This set of guidelines
-provides the following benefits:
+While BIDS was originally intended for Magnetic Resonance Imaging (MRI)
+data, it has recently been extended for electro-/magneto-encephalography (EEG/MEG)
+[@Niso2018, @Pernet2019], and intracranial EEG (iEEG) [@Holdgraf2019].
+In this context, many software packages for electrophysiology data analysis are
+now becoming BIDS compatible. BIDS is a rich system to curate, aggregate and
+annotate neuroimaging databases. For this reason we developed a dedicated
+software, ``MNE-BIDS`` with the goal to provide
+a programmable interface for BIDS datasets in electrophysiology. ``MNE-BIDS`` allows
```

Comment:
i think elsewhere, you've used neurophysiology as the catchall for all three

Reply:
what's the difference? :)